package.json

```diff
@@ -25,10 +25,13 @@
     "xml2js": "0.4.17"
   },
   "devDependencies": {
+    "eslint": "3.5.0",
+    "eslint-plugin-mocha": "4.5.1",
     "mocha": "3.0.2"
   },
   "scripts": {
-    "test": "mocha test"
+    "test": "mocha test",
+    "eslint": "node_modules/.bin/eslint ."
```

Comment:
Can we change this to not include the `node_modules` path. `npm run-script` will use it anyway.

reference: https://docs.npmjs.com/cli/run-script